Gems/EMotionFX/Code/Tests/UI/SkeletonOutlinerTestFixture.cpp

```diff
@@ -150,20 +160,42 @@ namespace EMotionFX
         // We did not crash, at least
     }
 
-    TEST_F(SkeletonOutlinerTestFixture, DISABLED_CopyAndPaste)
+    // This Test is disabled, because using AddColliderViaAddComponentButton is buggy and can only be called
+    // once
```

Comment:
update: Can this comment be removed now it's been enabled again? 😛

Reply:
But it's a part of history!